cli.js

```diff
@@ -31,13 +35,29 @@ const cli = meow(`
 	  $ boxen --border-style=doubleSingle …like everyone
 	  ╒══════════════╕
 	  │…like everyone│
+	  ╘══════════════╛
+
+		$ boxen --border-style=doubleSingle …like everyone
+	  ╒══════════════╕
+	  │…like everyone│
 	  ╘══════════════╛
 
 	  $ boxen --border-style='1234-~|║' ASCII ftw!
 	  1----------2
 	  |ASCII ftw!║
 	  3~~~~~~~~~~4
 
+	  $ boxen --align=center --width=16 Rainbows are so cool!
+	  ┌──────────────┐
+	  │ Rainbows are │
+	  │   so cool!   │
+	  └──────────────┘
+
+	  $ boxen --title='Yes it is' --title-alignment=center Is that a centered title?
```

Comment:
It's more readable to quote the text here.

```suggestion
	  $ boxen --title='Yes it is' --title-alignment=center 'Is that a centered title?'
```